packages/apps-config/src/endpoints/index.spec.ts

```diff
@@ -59,40 +59,6 @@ describe('urls are sorted', (): void => {
   });
 });
 
-describe('urls are not duplicated', (): void => {
-  let hasDevelopment = false;
-  let lastHeader = '';
-  const map = allEndpoints
-    .filter(({ isDisabled, isHeader, isUnreachable, text }): boolean => {
-      hasDevelopment = hasDevelopment || (!!isHeader && text === 'Development');
-
-      return !hasDevelopment && !isDisabled && !isUnreachable;
-    })
-    .reduce((map, { isHeader, text, value }): Record<string, string[]> => {
-      if (isHeader) {
-        lastHeader = text as string;
-      } else {
-        const path = `${lastHeader} -> ${text as string}`;
-        const key = value.endsWith('/')
-          ? value.substring(0, value.length - 1)
-          : value;
-
-        map[key] ||= [];
-        map[key].push(path);
-      }
-
-      return map;
-    }, {} as Record<string, string[]>);
-
-  Object
-    .entries(map)
-    .forEach(([url, paths]) =>
-      it(url, (): void => {
-        assert(paths.length === 1, `${url} appears multiple times - ${paths.map((p) => `\n\t"${p}"`).join('')}`);
-      })
-    );
-});
-
```

Comment:
Why did you need to remove this test entirely?

Reply:
Currently, this test is to prevent duplication of urls in the chain configuration of the app.
By the way, we need to add twice this url(wss://nodle-parachain.api.onfinality.io/public-ws).
Because we should customize the chain configuration. so I added nodle chains on top. and added that url on polkadot parachains.
As a result, duplicates with the same url.